pipeline/transformer/transformer_sierra/src/main/scala/uk/ac/wellcome/platform/transformer/sierra/transformers/SierraAgents.scala

```diff
@@ -35,14 +21,14 @@ trait SierraAgents {
       if (normalisePerson) {
         Person.normalised(
           label = label,
-          prefix = prefixString,
-          numeration = numeration
+          prefix = None,
+          numeration = None
```

Comment:
I thought the implication from Jonathan was that we were scrapping this field entirely, not just setting it to `None`?

(Note that doing so would change the index structure, so we'd need to do wellcometrust/platform#3547)

Reply:
Yea, this would be the first step in the transformers. I'd like to get the pipeline changed first, then the data (reindex) and then change the index shape.

Sound like a sensible plan?